src/Test.UnitTests.Sarif/Core/StackTests.cs

```diff
@@ -82,8 +84,9 @@ public void Stack_CreateFromAggregatedExceptionWithInnerException()
             {
                 File.Create(Path.GetInvalidFileNameChars()[0].ToString(), 0);
             }
-            catch (IOException exception)
+            catch (Exception exception)
```

Comment:
>Exception exception [](http://example.com/codeflow?start=19&length=19)

nit: may use this syntax if don't want to duplicate the exception handling code:

```
catch (Exception ex) when(ex is ArgumentException || ex is IOException)
{
     ...
}
```

Reply:
Yes! Nice and this resolves Ling's concern around the test code having a general Exception handler. There's benefit in *never* allowing that because that underscores to devs that it is never desirable in the product itself.